docker-services.yml

```diff
@@ -6,6 +6,8 @@ services:
     environment:
       - "INVENIO_ACCOUNTS_SESSION_REDIS_URL=redis://cache:6379/1"
       - "INVENIO_BROKER_URL=amqp://guest:guest@mq:5672//"
+      - "INVENIO_CELERY_BROKER_URL=amqp://guest:guest@mq:5672//"
+      - "INVENIO_QUEUES_BROKER_URL=amqp://guest:guest@mq:5672//"
```

Comment:
This is not needed since we are doing this https://github.com/zenodo/zenodo/blob/de48b2b81ecaad53481afe30fbbf8fd6b3493974/zenodo/config.py#L1298